### ISSUE TITLE
Wrong permissions in protobuf library

### ISSUE THREAD

**Issue by fansari** [romanz/trezor-agent#28]
Perhaps it should be mentioned in the the README (or better: be fixed) that the protobuf library comes with wrong permissions. Running as unprivileged user I got this error:

```
fansari@bat ~]$ trezor-agent raspi
2016-09-03 12:44:21,906 ERROR        Missing module: install via "pip install trezor"                                                     [factory.py:62]
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/trezor_agent/factory.py", line 50, in _load_trezor
    from trezorlib.client import TrezorClient, CallException
  File "/usr/lib/python2.7/site-packages/trezor-0.7.0-py2.7.egg/trezorlib/client.py", line 15, in <module>
    from . import mapping
  File "/usr/lib/python2.7/site-packages/trezor-0.7.0-py2.7.egg/trezorlib/mapping.py", line 1, in <module>
    from . import messages_pb2 as proto
  File "/usr/lib/python2.7/site-packages/trezor-0.7.0-py2.7.egg/trezorlib/messages_pb2.py", line 6, in <module>
    from google.protobuf.internal import enum_type_wrapper
ImportError: cannot import name enum_type_wrapper
2016-09-03 12:44:21,912 ERROR        Missing module: install via "pip install keepkey"                                                    [factory.py:79]
Traceback (most recent call last):
  File "build/bdist.linux-x86_64/egg/trezor_agent/factory.py", line 67, in _load_keepkey
    from keepkeylib.client import KeepKeyClient, CallException
  File "/usr/lib/python2.7/site-packages/keepkeylib/client.py", line 15, in <module>
    from . import mapping
  File "/usr/lib/python2.7/site-packages/keepkeylib/mapping.py", line 1, in <module>
    from . import messages_pb2 as proto
  File "/usr/lib/python2.7/site-packages/keepkeylib/messages_pb2.py", line 4, in <module>
    from google.protobuf.internal import enum_type_wrapper
ImportError: cannot import name enum_type_wrapper
2016-09-03 12:44:21,917 ERROR        Connection error: 0 devices found

```

A workaround is fixing the probobuf permsisions manually:

`chmod  -R o+r /usr/lib/python2.7/site-packages/protobuf-3.0.0-py2.7.egg`

See also here:
[https://github.com/google/protobuf/issues/737](https://github.com/google/protobuf/issues/737)

**Comment by romanz** [comment]
Thanks for reporting this issue!
I will keep it here and update the documentation.

**Comment by romanz** [comment]
Seems to be fixed at protobuf 3.4.0:
```bash
$ pip3 freeze | grep protobuf
protobuf==3.4.0
$ ls -l /home/roman/.local/lib/python3.5/site-packages/google/protobuf/internal/enum_type_wrapper.py 
-rw-r--r-- 1 roman roman 3554 Oct  9 16:14 /home/roman/.local/lib/python3.5/site-packages/google/protobuf/internal/enum_type_wrapper.py
```
